utilities/_border.scss

```diff
@@ -53,30 +53,28 @@
 
 @if $u-classes-border == true {
 
-  $border-values: none;
+  @each $border-width, $border-width-value in $border-width-values {
```

Comment:
Not an issue with the code, but would you mind explaining this part briefly? I'm still wrapping my head around some of the SCSS functionality and I'm not sure exactly what is being referred to here.